lib/openpgp_crypt_rsa.php

```diff
@@ -245,8 +246,18 @@ static function convert_key($packet, $private=false) {
     $rsa = self::crypt_rsa_key($mod, $exp);
 
     if($private) {
-      if($packet->key['p'] && $packet->key['q']) $rsa->primes = array($packet->key['p'], $packet->key['q']);
-      if($packet->key['u']) $rsa->coefficients = array($packet->key['u']);
+        /**
+         * @see https://github.com/phpseclib/phpseclib/issues/1113
+         * Primes and coefficients now use BigIntegers.
+         **/
+        //set the primes
+        if($packet->key['p'] && $packet->key['q'])
+            $rsa->primes = array(
+                1=> new BigInteger($packet->key['p'], 256),
+                new BigInteger($packet->key['q'], 256)
+            );
+        // set the coefficients
+        if($packet->key['u']) $rsa->coefficients = array(2 => new BigInteger($packet->key['u']));
```

Comment:
looks like the tests point out that it does. Not sure why the tests didn't catch that on my machine.